themes/theme-default/src/client/styles/code.scss

```diff
@@ -229,18 +229,12 @@ div[class*='language-'] {
       padding-top: 1.3rem;
       font-size: 0.875em;
       line-height: $line-height;
-      counter-reset: line-number;
 
       .line-number {
         position: relative;
         z-index: 3;
         user-select: none;
         height: #{$line-height}em;
```

Comment:
can we just set it to the number?

Reply:
Originally intended to use `counter-reset: line-number attr(data-start number)`,
but due to its poor compatibility, the current approach was chosen.

I could directly write the style into `.line-numbers` instead of using `data-start`.

Reply:
nope, I was talking about `height: #{$line-height}em;` to be `line-height: #{$line-height};`